.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 * @mozilla-services/autograph-devs @mozilla-services/autograph-mergers
 
-signer/xpi/ @mozilla-services/autograph-devs @mozilla-services/autograph-mergers @mozilla/addons-engineering
\ No newline at end of file
+signer/xpi/ @mozilla-services/autograph-devs @mozilla-services/autograph-mergers @mozilla/addons-amo-engineering
```

Comment:
Ugh, this is telling us that (like my last PR) this CODEOWNERS has an error around addons-amo-engineering. The last time, we still needed to give `@mozilla-services/addons-engineering` the ability to `Write` to this repo.

This time we have to give this team `Write` access, but I can't seem to find a team that matches `@mozilla/addons-amo-engineering` in the search functionality to give them access. I suspect either that team is not a public team, or GitHub isn't allowing me to add anyone from there because I'm not a member of any team in the `@mozilla` org (contrast that with `@mozilla-services`). I'm not able to confirm that `@mozilla/addons-amo-engineering` even exists because I'm not on any team in `@mozilla`.

We may need to engage with the addons team directly to see what exact team they're using and if they can make that team public.

Reply:
Either use the team from the mozilla-services org OR from the mozilla org:

1. https://github.com/orgs/mozilla-services/teams/addons-engineering
2. https://github.com/orgs/mozilla/teams/addons-amo-engineering

Reply:
Okay, I guess we'll use the one in the same org, then since the other one seems to not be marked as `Public`?